app/controllers/api/v3/timelines/for_you_controller.rb

```diff
@@ -138,33 +50,10 @@ def account_from_acct
     Account.where(username: username, domain: domain).first
   end
 
-  def user_from_param
-    PersonalForYou.new.user(acct_param)
-  end
-
   def acct_param
     params.require(:acct)
   end
 
-  def for_you_params
-    params.permit(
```

Comment:
It feels weird to remove this, but not have it elsewhere. Is it really no longer needed? Just confirming.

Reply:
they are no longer needed on the v3 endpoint, but are very much on and needed on the v4 endpoint [here](https://github.com/TheBLVD/moth.social/pull/219/files/837888e6786e5b313252db5ced89f3a6985d9365#diff-cf6c2eeefe81a4051f6cef1eb3928b34bebd531d2eac8cc9587860bf637dfe39L130)